Sources/PackageGraph/PackageGraphRoot.swift

```diff
@@ -0,0 +1,28 @@
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import Basic
+import PackageModel
+
+/// Represents the inputs to the package graph.
+public struct PackageGraphRoot {
+    
+    /// The list of root manifests.
+    public let manifests: [Manifest]
+
+    /// The top level dependencies.
+    public let dependencies: [RepositoryPackageConstraint]
```

Comment:
It is the same but it is the data structure which is used by the resolver as input (it is not exposed to PackageDescription clients). This will need to be replaced by a new struct because we need a location string.